app/components/show/catalog_record_id_component.html.erb

```diff
@@ -1,8 +1,8 @@
-<%= catkey %>
+<%= catalog_record_id %>
 
 <% if allows_modification? %>
-  <%= link_to edit_item_catkey_path(item_id: id),
-              aria: { label: 'Manage catkey' },
+  <%= link_to edit_item_catalog_record_id_path(item_id: id, folio: params[:folio]),
```

Comment:
The `folio` param here allows easier testing of the changes in this PR by tossing `?folio=true` at the Argo show view URL. There are a few similar changes below to carry this param through to other views in the catalog record ID editing flow.

Reply:
They do not at this point, so 🕺🏻